demo/chartTypes.ts

```diff
@@ -113,6 +113,16 @@ let chartGroups = [
         ]
       },
       {
+        name: 'Line Chart with Ranges',
```

Comment:
Should this be a different chart type? I think it should just be an option (`Show min-max range` - boolean) on the line chart, under data. Selecting it would modify the data to just add the min and max values. This would clean up the demo page a bit.